spring-data-microservice-transaction-sample/customer-service/src/main/java/sample/customer/CustomerService.java

```diff
@@ -78,7 +90,7 @@ public void getCustomerInfo(
       backoff = @Backoff(delay = 1000, maxDelay = 8000, multiplier = 2))
   @Override
   public void repayment(RepaymentRequest request, StreamObserver<Empty> responseObserver) {
-    execAndReturnResponse(responseObserver, "Repayment", () ->
+    execAndReturnResponse("Repayment", () ->
```

Comment:
I changed the order of the arguments of `execAndReturnResponse()` to be consist with `execOperations()` in `microservice-transaction-sample`.